Drop-In/src/main/java/com/braintreepayments/api/DropInInternalClient.java

```diff
@@ -148,10 +148,8 @@ void shouldRequestThreeDSecureVerification(PaymentMethodNonce paymentMethodNonce
     void tokenizePayPalRequest(FragmentActivity activity, PayPalFlowStartedCallback callback) {
         PayPalRequest paypalRequest = dropInRequest.getPayPalRequest();
         if (paypalRequest == null) {
-            paypalRequest = new PayPalVaultRequest();
+            paypalRequest = new PayPalVaultRequest(false);
         }
-        // TODO: set hasUserLocationConsent on paypalRequest
-        dropInRequest.hasUserLocationConsent();
```

Comment:
`hasUserLocationConsent` is set on the `PayPalRequest` directly. No need to set it here.

Reply:
If the merchant has set `hasUserLocationConsent` on their `dropInRequest` already - can we pass that value in when creating the default `PayPalVaultRequest`?

Reply:
yep, great suggestion! i'll update this.

unrelated question - if this function gets called with a null `PayPalRequest` and a default one is created, are there scenarios where `payPalClient.tokenizePayPalAccount()` will ever succeed? are there required fields on the request?

Reply:
Before this change, there were no required fields for a `PayPalVaultRequest` (checkout request requires amount). So if a merchant doesn't pass a custom `PayPalRequest` to their `DropInRequest`, drop-ins default behavior is the PayPal billing agreement/vault flow - it should succeed when we create the default vault request.